quodlibet/ext/query/conditional.py

```diff
@@ -15,7 +15,7 @@ class ConditionalQuery(QueryPlugin):
     PLUGIN_NAME = _("Conditional Query")
     PLUGIN_DESC = _("Chooses the query to match based on a condition query.")
     key = 'if'
-    usage = "<b><tt>@(if: condition-query, then-query, else-query)</tt></b>"
+    usage = "<b><tt>" + _("@(if: condition-query, then-query, else-query)") + "</tt></b>"
```

Comment:
Hmm seeing the changes below, might be worth a helper? (I should have done this originally...)
```python
def markup_for_syntax(text: str) -> str
    return f"<b><tt>{util.escape(text)}</tt></b>"
```

Reply:
I have added the helper.